ts/src/kraken.ts

```diff
@@ -231,6 +232,7 @@ export default class kraken extends Exchange {
                 'delistedMarketsById': {},
                 // cannot withdraw/deposit these
                 'inactiveCurrencies': [ 'CAD', 'USD', 'JPY', 'GBP' ],
+                'sliceAllowedForCurrencyIds': [],
```

Comment:
@ttodua what's this?

Reply:
ah, remnant from previous commit, sry, removing it 1 sec